liquibase-core/src/main/java/liquibase/integration/spring/SpringResource.java

```diff
@@ -3,6 +3,7 @@
 import liquibase.exception.UnexpectedLiquibaseException;
 import liquibase.resource.OpenOptions;
 import org.springframework.core.io.Resource;
+import org.springframework.core.io.UrlResource;
```

Comment:
this looks like unused import

Reply:
Hi @konikvranik,

Thanks for pointing this out. I'm going to remove this unused import in a different PR.